pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
What about the input of `np.uint`?

This seems a little tricky because on the one hand, it would indicate we would want the smallest unsigned integer possible, but because Python has no built-in `uint`, it will be cast to `np.uint64` or `np.uint32` (depending on architecture), making it appear that the user specified one of those.

Could go either way, but would like to make that clear in documentation.